internal/core/metadata/operators/command/get.go

```diff
@@ -21,3 +23,30 @@ func NewDeviceIdExecutor(db CommandLoader, deviceId string) DeviceIdExecutor {
 func (op commandLoadByDeviceId) Execute() (commands []contract.Command, err error) {
 	return op.database.GetCommandsByDeviceId(op.deviceId)
 }
+
+//Get All commands
+type CommandAllExecutor interface {
```

Comment:
Can we add a test suite(test table) for this new operator?  I see that the tests in `rest_command_test.go` cover all the lines in this file, but I still think it would be nice to unit test this operator in isolation to get better functional coverage.

Reply:
@difince To zero in on what @AnthonyMBonafide is referring to

1.) run `make test`
2.) run `go tool cover -html=coverage.out`
3.) Once the browser opens, use the drop-down at the top to view `internal/core/metadata/operators/command/get.go` 

It should have %20 coverage with this PR and the untested `commandLoadAll` logic is in red. Once the logic is covered, re-run the above steps and the lines will be green.